### ISSUE TITLE
ETH to LTC Translation is Incorrect on Purchase

### ISSUE THREAD

**Issue by jjeffryes** [OpenBazaar/openbazaar-desktop#1840]
This is happening on Dispute payout precision #1838 and ethereum-master latest as of 11-27-19.

I made a purchase of an item at a price of 0.0000002 ETH in LTC (the item price is 0.0000000001, the quantity was 2000).

The receipt on the purchase page showed a total of 0.00000064 LTC, which seems right.

But the purchase call had an amount of 314000, or 0.00314 LTC when it was submitted to the server.

**Comment by rmisio** [comment]
> But the purchase call had an amount of 314000, or 0.00314 LTC when it was submitted to the server.

I think Im confused. The client does not provide an amount to the server in the purchase call. Are you referring to some value the server is returning? If so, wouldn't that be a server bug?

**Comment by jjeffryes** [comment]
Yes, sorry. I mixed up the headers and response, and forgot we just send the quantity to the server.
